addons/bestja_frontend_ucw/static/src/css/custom.css

```diff
@@ -126,27 +126,34 @@ h4, .h4{
     color: #FFFFFF;
 }
 
-.violet_section{
-    background: url('http://bestja.d3.laboratorium.ee/img/zostan_wolontariuszem1200x413.jpg') no-repeat 50% 0%;
-    background-size: 150% auto;
+.for_organizations_first_section{
+    background: url('http://bestja.d3.laboratorium.ee/img/1920/dlaorganizacji.jpg') no-repeat 50% 0%;
     color: #FFFFFF;
 }
 
-.violet_section_about_us{
-    background: url('http://bestja.d3.laboratorium.ee/img/1920/onas_1920x400.jpg') no-repeat 50% 0%;
-    background-size: 140% 140%;
+.become_volunteer_first_section{
+    background: url('http://bestja.d3.laboratorium.ee/img/1920/zostan_wolontariuszem1.jpg') no-repeat 50% 0%;
     color: #FFFFFF;
 }
 
-.violet_section_development{
-    background: url('http://bestja.d3.laboratorium.ee/img/medium/zostan_wolontariuszem2_1920x434.jpg') no-repeat 50% 0%;
-    background-size: auto 120%;
+.become_volunteer_qualifications_section{
+    background: url('http://bestja.d3.laboratorium.ee/img/1920/zostan_wolontariuszem2.jpg') no-repeat 50% 0%;
     color: #FFFFFF;
 }
 
-.violet_section_join_us{
-    background: url('http://bestja.d3.laboratorium.ee/img/medium/zostan_wolontariuszem3.jpg') no-repeat 50% 0%;
-    background-size: auto auto;
+.become_volunteer_join_us_section{
+    background: url('http://bestja.d3.laboratorium.ee/img/1920/zostan_wolontariuszem3.jpg') no-repeat 50% 0%;
+    color: #FFFFFF;
+}
+
+.violet_section{
```

Comment:
Class names should not describe *appearance* of an element, especially its colour - because it is likely to change in the future (and you will end up with a green section named "violet"). The names should be based on a *function* of the element.